mage/releases/publish.go

```diff
@@ -72,17 +73,15 @@ exec echo "$GITHUB_TOKEN"
 	pwd, _ := os.Getwd()
 	script := filepath.Join(pwd, askpass)
 
-	must.Command("git", "config", "user.name", "Porter Bot").In(dir).RunV()
-	must.Command("git", "config", "user.email", "bot@porter.sh").In(dir).RunV()
 	must.Command("git", "config", "core.askPass", script).In(dir).RunV()
 }
 
 func publishPackage(pkgType string, name string) {
 	mg.Deps(tools.EnsureGitHubClient, ConfigureGitBot)
 
-	info := mage.LoadMetadata()
+	info := LoadMetadata()
 
-	repo := os.Getenv("PORTER_RELEASE_REPOSITORY")
+	repo := os.Getenv(ReleaseRepository)
```

Comment:
Wondering if we want to specifically error out if repo is empty (similarly for `publishPackageFeed` below and its `remote` value)?  Or, will it be obvious to the user that an empty value (via an unset env var) is to blame for this target not working?